examples/analytics.amp.html

```diff
@@ -237,6 +237,29 @@
 </amp-analytics>
 <!-- End comScore example -->
 
+<!-- INFOnline example
+
+  Please note: the url variable must point to an HTML file including the ivw
```

Comment:
Capitalize: IVW. Can you also include a url for clarity? So that would make the whole suggestion to be to add "IVW (http://www.ivw.de)".